internal/validate/validate.go

```diff
@@ -158,3 +158,18 @@ func DBUsername(val interface{}) error {
 
 	return fmt.Errorf("%w: %s", ErrInvalidDBUsername, name)
 }
+
+var ErrWeakPassoword = errors.New("the password provided is too weak, as it can be found in most commonly used password lists")
```

Comment:
[nit] this is two long, for sentinel errors you want them short
I would make it (but feel free to improve on)
```go
var ErrWeakPassoword = errors.New("invalid password")

func WeakPassword(val interface{}) error {
...
if commonPasswords[password] {
	return fmt.Errorf("%w: password is too common", ErrWeakPassoword)
}
```
don't return the password when you asked it masked just to expose it in an error